Project-1/README.md

```diff
@@ -10,113 +10,324 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+
   ```(a) (A→B)→C
   (b) (A→(B∨C))∨(C→¬A)
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
-``` 
+
+``` (a) (A ∨ C) ∧ (¬B ∨ C)
+(b) B ∨ ~A Tautology should have form of p v ~p
+(c) B v ~B Tautology  
+
+used CNF calculator https://www.artima.com/cs/cnf.html & https://www.erpelstolz.at/gateway/formular-uk-zentral.html
+
 
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+
   ```(a)  B(x,y) that says that x is a brother of y
   (b)  A(x,y) that says that x is an aunt of y
   (c)  C(x,y) that says that x and y are cousins 
   (d)  O(x) that says that x is an only child
   (e)  T(x) that says that x has exactly two brothers 
-```
+
+```(a) B(x,y) = ∃u(P(u,x)∧P(u,y))∧¬F(x)∧¬(x=y) - There is some x and some y and some z such that z is the parent of y and z is the parent x and x is not female
+(a) A brother is an individual who identifies as male and has at least one sibling that is, they share at least one parent. Half-siblings and step-siblings can be brothers. In the example above, x would be an individual who shares at least one parent with y and x would identify as male.
+
+(b) A(x,y) = ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x) - There is some w and some x and some y and some z such that z is the parent of y and w is the parent of z and w is the parent of x and x is female
+(b) An Aunt is a female identifying sibling of a parent. In the example above x would be an Aunt of y so long as x is the child of y's sibling
+
+(c) C(x,y) = ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x= - There is some v and some w and some x and some y and some z such that v is the parent of w and v is the parent of z and w is the parent of x and z is the parent of y
+(c) A cousin is a relationship between two individuals whose parents are siblings. Such that you have a cousin in the child of your aunt
+
+(d) ∃x∃y∀z(P(y,z)→z=x) - There is some x and some y such that for all z if y is the parent of z then z and x are identical 
+(d) An only child is an individual who was raised in a house primarily without siblings. Individuals who have step or half siblings but were raised pimarily alone are only children. 
+
+(e)∃v∃w∃x∃y∀z(P(v,x)∧P(v,w)∧¬Fw∧P(v,y)∧¬Fy∧∀z((P(v,z)∧¬Fz)→((z=x)∨(z=w)∨(z=y)))∧v≠w∧v≠x∧v≠y∧v≠z)  - There is some v and some w and some x and some y such that for all z if v is the parent of x and v is the parent of w and w is not female and v is the parent of y and y is not female and for all instances of z v is a parent of z and z is not female ten z is identical with w or z is identical with w or z is identical with y
```

Comment:
Change from "∃v∃w∃x∃y∀z" to "∃v∃w∃y"

In "... ∧v≠w∧v≠x∧v≠y∧v≠z", delete "∧v≠z"